exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
+library(tidyverse)
+library(ggthemes)
```

Comment:
I see that you are loading the `ggthemes` package here.

But when I look at your code, you are not using any functions from `ggthemes`. So, why load it?